src/mame/drivers/missbamby.cpp

```diff
@@ -11,7 +11,7 @@ Known games on this hardware:
  | YES    | Lucky Player      | Cirsa              | Cirsa PCB 810702 A |
  | YES    | Miss Bamby        | Automatics Pasqual |                    |
  | YES    | Golden Winer      | Reben              |                    |
```

Comment:
Is this "Golden Winer" (something involving wine?) or "Golden Winner"?

Reply:
It's a typo, fixed!